src/base/server_type.cpp

```diff
@@ -86,11 +86,6 @@ void finalize_server_type ()
     {
       ats_Gl.disconnect_page_server ();
     }
-  else
-    {
-      assert (get_server_type () == SERVER_TYPE_PAGE);
-      ps_Gl.disconnect_active_tran_server ();
-    }
```

Comment:
This might still have to be done. If the page server is shutting down, it will have to remove the connection with the transaction server.

What is the reasoning for removing the code?

Reply:
If one PS shuts down shouldn't the ATS be notified? The connection from the ATS point of view is still up